models/marts/dim_customers.sql

```diff
@@ -0,0 +1,36 @@
+{{ config(materialized='table') }}
+
+with customers as (
+    select * from {{ ref('stg_customers') }}
+),
+orders as (
+    select * from {{ ref('stg_orders') }}
```

Comment:
You can actually pull from `dim_orders` already where you do some cleanup so you don't have to do the same joins all over again :) 
```suggestion
    select * from {{ ref('dim_orders') }}
```